Server/makefile

```diff
@@ -5,24 +5,29 @@ OBJS  = NiKServer.o NiKHost.o NiKHost2.o NiKHost3.o NiKVersion.o NodeComm.o \
 
 all: NiKServer
 
-setup:
-  makedir Debug Debug/$(CPU)
-  makedir NoDebug NoDebug/$(CPU)
+Debug:
+  makedir Debug
+
+Debug/$(CPU): Debug
+  makedir Debug/$(CPU)
+
+NoDebug:
+  makedir NoDebug
+
+NoDebug/$(CPU): NoDebug
+  makedir NoDebug/$(CPU)
 
 clean:
   delete \#?.o
-  delete Debug/$(CPU)/\#? NoDebug/$(CPU)/\#?
+  delete Debug NoDebug all
 
-NiKServer: $(OBJS)
-  slink FROM lib:c.o $(OBJS) \
-        TO Debug/$(CPU)/NiKServer \
-        LIB lib:sc.lib+lib:amiga.lib+/UtilLib/Debug/$(CPU)/nikomutils.lib \
-        NOALVS NOICONS
+NiKServer: Debug/$(CPU) NoDebug/$(CPU) $(OBJS)
+  slink FROM lib:c.o $(OBJS) TO Debug/$(CPU)/NiKServer \
+        LIB lib:sc.lib+lib:amiga.lib+/UtilLib/Debug/$(CPU)/nikomutils.lib NOICONS
   slink FROM Debug/$(CPU)/NiKServer TO NoDebug/$(CPU)/NiKServer NODEBUG NOICONS
 
-
 NiKServer.o: NiKServer.c /Include/NiKomStr.h
-  sc $(FLAGS) NiKServer.c
+  sc $(FLAGS) DEFINE=NIKRELEASE=$(NIKRELEASE) NiKServer.c
```

Comment:
I just removed the "DEFINE=NIKRELEASE=$(NIKRELEASE)" in a change yesterday. (I had missed this one when moving all release labels from the makefiles to an include file.) Did you get a merge conflict and managed to put it back? (Or maybe it's because you started over with your repo and copied files from the old repo. Copying files around is always dangerous when working with version control systems.)